README.md

```diff
@@ -1,6 +1,6 @@
 # Lock Nessie
 
-**Easily integrate OpenID authentication and authorization into your Apache Nessie and Iceberg projects.**
+**Easily integrate OpenID authentication and authorization into your [Project Nessie](https://projectnessie.org/) and Iceberg systems.**
```

Comment:
Apache Iceberg should be used there (for trademark compliancy).

Reply:
Good point, @jbonofre , fixed!